app/partials/errorDetail.html

```diff
@@ -0,0 +1,20 @@
+<table class="detail-table" ng-if="error">
+    <thead>
+        <tr>
+            <th class="id" scope="col">Loan/Application Number</th>
```

Comment:
The header should change based on whether you're displaying the loanNumber or lineNumber?